lib/BiochemistryAPI/BiochemistryAPIImpl.py

```diff
@@ -22,58 +23,11 @@ class BiochemistryAPI:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "0.1.1"
-    GIT_URL = "git@github.com:kbaseapps/BiochemistryAPI.git"
-    GIT_COMMIT_HASH = "b82efb604f55537e62cdf2bc8960369a8c640d27"
+    VERSION = "0.1.2"
+    GIT_URL = "https://github.com/kbaseapps/BiochemistryAPI.git"
+    GIT_COMMIT_HASH = "15eb8ad3e8aa95eb2f632cbfe0b96199c50e97c5"
 
     #BEGIN_CLASS_HEADER
-    @staticmethod
```

Comment:
Was all of this stuff dead code?

Reply:
No, I just refactored by moving it to the utils file. Sorry to mix refactoring & new features on the same PR